### ISSUE TITLE
Customer Update Password Error via Profile UI on v1.2.4

### ISSUE THREAD

**Issue by rickyprat** [polonel/trudesk#515]
**Is this a BUG REPORT or FEATURE REQUEST?**:

- [x] BUG 
- [ ] FEATURE

**What happened**: After upgrading to v1.2.3 and then v1.2.4, I read this:
Bug Fixes
- accounts: security enhancement ([83fd5a8](https://github.com/polonel/trudesk/commit/83fd5a8))

So, I tried login as a Customer, then I changed the Name, Password, Email, and it appeared to be successful. When I logout and login with the new password, it said Invalid Username/Password. But I can login using the old password. After that, I can confirm that the Name and Email has been updated. Seems like only the password is not updated with this method.

![v1 2 4 Customer Update Profile Appeared Success](https://user-images.githubusercontent.com/108453512/176642867-11aca5eb-580b-4867-9336-b03d46132bc2.png)

**What did you expect to happen**: User (Customer) should be able to update their profile (Name, Password, Email) via UI. 

**How to reproduce it (as minimally and precisely as possible)**: Login as a Customer, click Profile, change the Name, Password, Email. Then logout and login again with the new password.

**Anything else we need to know?**: No error on output.log

**Environment**:
- Trudesk Version: 1.2.4
- OS (e.g. from /etc/os-release):  Docker Container on Debian GNU/Linux 11
- Node.JS Version: 16.14.2
- MongoDB Version: 5.0.9
- Is this hosted on cloud.trudesk.io: No

**Comment by polonel** [comment]
Does this occur only on the customer account? Does the admin account password change as expected?

**Comment by rickyprat** [comment]
It only occurs on Customer account.

- Customer Change their own password = NOT OK
- Admin Change Customer password = OK
- Admin Change other Admin/Agent password = OK
- Admin/Agent Change their own password = OK

**Comment by polonel** [comment]
The profile section has been completely revamped in the upcoming `v1.2.5`. I will check the other errors you are experiencing on `v1.2.4`. There were a lot of account-based security fixes/enhancements. 

One last thing to check/test if you don't mind. (My codebase is a mess right now with the new version)

**Try to disable the password complexity requirement in `settings>accounts>password complexity`**

This new enhancement was added in `v1.2.4`  and was enabled by default.

**Comment by rickyprat** [comment]
I checked v1.2.3 and v1.2.4, the option is there and enabled by default.

When the password complexity was enabled (default) and I tested either versions to update from /profile, never got an error about password, same result whether the password meet or didn't meet the requirement (less than 8 character).

On v1.2.4, I tried disabling the option and got the same result on all accounts (Admin, Agent, Customer).
